src/cluster_asm.c

```diff
@@ -382,6 +425,52 @@ void clusterMigrationCommand(client *c) {
     }
 }
 
+void asmImportFailed(asmTask *task) {
```

Comment:
as we discussed, lets think about if we extend this to save error message to list in CLUSTER MIGRATION STATUS. Otherwise, I have a feeling that it will invite a lot of questions to us once other teams start using it. Checking out logs are quite convenient for us but I realized it'd be a lot better for other teams if we can put that into the command output. (we can do that later ofcourse, not part of this pr)

Reply:
yes, totally agree, i just make a PR when i found the changed lines is not small, a suitable size PR is easy to review and to avoid conflicts.

I will do it later